api/bin/chainflip-lp-api/src/main.rs

```diff
@@ -96,7 +96,8 @@ pub trait Rpc {
 	async fn request_liquidity_deposit_address(
 		&self,
 		asset: RpcAsset,
-	) -> Result<String, AnyhowRpcError>;
+		wait_for: WaitFor,
```

Comment:
Not sure the rpc macro, will generate this as an optional arg, just because it impls default. If that was what you intended